lib/app/modules/alarmRing/views/alarm_ring_view.dart

```diff
@@ -151,6 +151,58 @@ class AlarmControlView extends GetView<AlarmControlController> {
                     ],
                   ),
                 ),
+                Obx(
+                  () => Visibility(
+                    visible:
+                        controller.currentlyRingingAlarm.value.note.isNotEmpty,
+                    child: (Container(
+                      padding: EdgeInsets.all(16.0),
+                      decoration: BoxDecoration(
+                        color: themeController.isLightMode.value
+                            ? kLightPrimaryBackgroundColor
+                            : kprimaryBackgroundColor,
+                        borderRadius: BorderRadius.circular(10.0),
+                        boxShadow: [
+                          BoxShadow(
```

Comment:
Everything looks good, but I disagree with the way the note should be displayed on the alarm screen. The big bold Note should definitely be removed, the box shadow looks very weird since we haven't used it anywhere else.

The actual note itself - We can center it? Maybe italics (whatever looks better). But the big "Note" heading is definitely offputting imo.